frontend/public/components/utils/resource-log.jsx

```diff
@@ -247,17 +299,20 @@ export class ResourceLog extends SafetyFirst {
           Refresh
         </button>
       </p>}
-      <LogControls
-        dropdown={dropdown}
-        status={status}
-        toggleStreaming={this._toggleStreaming}
-        onDownload={this._download} />
-      <LogWindow
-        lines={lines}
-        linesBehind={linesBehind}
-        bufferFull={bufferFull}
-        status={status}
-        updateStatus={this._updateStatus} />
+      <div className="log-console">
+        <LogControls
```

Comment:
nit: I think I missed this element when I first created this, but I try to keep the props in alphabetical order when they are split across multiple lines like this. This isn't consistent across the repo, but when I encounter them in something I work on, I try to fix it. I don't think it's all that important, but just thought I'd mention it.

Reply:
fixed